File: demos/demo8.py
#!/usr/bin/python2.7
import sys
while (1):
	sys.stdout.write (str("Enter a number: \n"))
	help = raw_input()
	help = str(help)[:-1] if str(help)[-1] == '\n' else help
	if (float(help) % float(4) != 0):
		sys.stdout.write (str(""+str(help)+" is not divisible by 4\n"))
		continue
	
	else :
		sys.stdout.write (str(""+str(help)+" is divisible by 4\n"))
		break
	

sys.stdout.write (str("And we are done!\n"))

File: demos/demo8.pl
#!/usr/bin/perl

while (1){
	print "Enter a number: \n";
	$help = <STDIN>;
	chomp($help);
	if ($help % 4 != 0){
		print "$help is not divisible by 4\n";
		next;
	} else {
		print "$help is divisible by 4\n";
		last;
	}
}
print "And we are done!\n";
